src/main/scala/cromwell/binding/WdlNamespace.scala

```diff
@@ -313,14 +293,33 @@ case class WdlBinding(ast: Ast, source: WdlSource, importResolver: ImportResolve
     }
     val taskName = sourceString(ast.getAttribute("task"))
     val task = findTask(taskName) getOrElse invalidTask
-    val inputs = WdlBinding.getCallInput(ast)
+    val inputs = processCallInput(ast)
     new Call(alias, taskName, task, inputs.toMap, this)
   }
 
+  private def processCallInput(ast: Ast): Map[String, WdlExpression] = AstTools.getCallInputAsts(ast).map {a =>
+    val key = sourceString(a.getAttribute("key"))
+    val expression = new WdlExpression(a.getAttribute("value"))
+    (key, expression)
+  }.toMap
+
+  private def processWdlType(ast: AstNode): WdlType = {
+    ast match {
+      case t: Terminal if t.getSourceString == WdlFileType.toWdlString => WdlFileType
+      case t: Terminal if t.getSourceString == WdlStringType.toWdlString => WdlStringType
+      case t: Terminal if t.getSourceString == WdlIntegerType.toWdlString => WdlIntegerType
+      case t: Terminal if t.getSourceString == WdlFloatType.toWdlString => WdlFloatType
+      case t: Terminal if t.getSourceString == WdlBooleanType.toWdlString => WdlBooleanType
+      case t: Terminal if t.getSourceString == WdlObjectType.toWdlString => WdlObjectType
```

Comment:
Hmm dunno if @geoffjentry is okay with it but I think it's probably a good idea.